bin/lib/ce_install.py

```diff
@@ -249,19 +280,18 @@ def amazon_check():
 @click.argument("filter_", metavar="FILTER", nargs=-1)
 def squash(context: CliContext, filter_: List[str], force: bool, image_dir: Path):
     """Create squashfs images for all targets matching FILTER."""
-    for installable in context.get_installables(filter_):
-        if not installable.is_installed():
-            _LOGGER.warning("%s wasn't installed; skipping squash", installable.name)
-            continue
-        destination = image_dir / f"{installable.install_path}.img"
-        if destination.exists() and not force:
-            _LOGGER.info("Skipping %s as it already exists at %s", installable.name, destination)
-            continue
-        if installable.nightly_like:
-            _LOGGER.info("Skipping %s as it looks like a nightly", installable.name)
-            continue
-        _LOGGER.info("Squashing %s to %s", installable.name, destination)
-        installable.squash_to(destination)
+
+    with context.pool() as pool:
+        for installable, destination in [
```

Comment:
is it possible to put the array in a variable or preferable a function or something, this is a little bit crazy to read imo

Reply:
Sure thing! I'm a bit happy to do these on one big line, but _I_ know what I was thinking at the time :) Will make the change, thanks for the feedback!